### DISCUSSION TITLE
Backup my old database is not working

### DISCUSSION THREAD

**Discussion by SocialCodeClub** [supabase/supabase#27627] [Questions]
I am following this guide https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#backup-your-old-database to create a backup for my old data base.

However when I run this command: `supabase db dump --db-url "$my_url" -f data.sql --use-copy --data-only` I get the following error: `Dumping data from remote database...
pg_dump: error: connection to server on socket "/private/tmp/.s.PGSQL.5432" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
error running container: exit 1
`

In the past I was able to do that successfully, but now I run into this error. Can you please help me?

**Comment by beagreenlite** [comment]
any updates on this?

**Reply by mu-jchung** [reply]
The solution for me was to run the "supabase db dump ..." command as root. This was on a Debian 12/13 system.
